internal/diagnostics/executable.go

```diff
@@ -39,7 +39,7 @@ func diagnoseExecutable(cfg *config.Config, cliVersion string, opts ExecutableOp
 		group.AddMsg(
 			failed,
 			fmt.Sprintf("Version does not meet minimum of %s required by catalog", style.Code(cfg.MinVersion)),
-			msg(hint, fmt.Sprintf("Update joy using: %s", style.Code("brew upgrade joy"))),
+			msg(hint, fmt.Sprintf("Update joy using: %s", style.Code("brew update && brew upgrade joy"))),
```

Comment:
@alexstojda I thought you mentioned `brew update` was no longer required, so I updated the different READMEs accordingly, but I can't find any docs supporting that... 🤔

Reply:
Yeah I thought it was the case, and do recall seeing it somewhere, but I too can no longer find docs supporting it. 

Perhaps it's not the case for 3rd party taps?